src/timer/Timer.js

```diff
@@ -1,17 +1,19 @@
-function Timer(seconds) {
-  this.seconds = seconds;
-}
+class Timer {
+  constructor(seconds) {
+    this.seconds = seconds;
+  }
 
-Timer.prototype.start = function () {
-  var instance = this;
-  var timerInterval = setInterval(function () {
-    if (instance.seconds === 0) {
-      clearInterval(timerInterval);
-    }
+  start() {
+    let instance = this.seconds;
```

Comment:
> Refactor the code to avoid assigning this to a variable.

`instance` on line 9 could be replaced with `this.seconds`. Although this technically meets the requirement in the README.